lib/spacesuit.ex

```diff
@@ -10,23 +10,24 @@ defmodule Spacesuit do
   @http_port 8080
 
   def start(_type, _args) do
-    Logger.info "Spacesuit starting up on :#{@http_port}"
+    Logger.info("Spacesuit starting up on :#{@http_port}")
 
-    dispatch = Spacesuit.Router.load_routes |> :cowboy_router.compile
+    dispatch = Spacesuit.Router.load_routes() |> :cowboy_router.compile()
 
-    {:ok, _} = :cowboy.start_clear(
-        :http, 100, [port: @http_port],
-        %{
-           env: %{
-            dispatch: dispatch
-           },
-           middlewares: [
-             #:cowboy_router, Spacesuit.DebugMiddleware, Spacesuit.AuthMiddleware, :cowboy_handler
-             :cowboy_router, Spacesuit.CorsMiddleware, Spacesuit.AuthMiddleware, :cowboy_handler
-           ]
-         }
-    )
+    {:ok, _} =
+      :cowboy.start_clear(:http, 100, [port: @http_port], %{
+        env: %{
+          dispatch: dispatch
+        },
+        middlewares: [
+          # :cowboy_router, Spacesuit.DebugMiddleware, Spacesuit.AuthMiddleware, :cowboy_handler
```

Comment:
Do we need to keep these commented middlewares?

Reply:
It prevents having to remember where to put the debug middleware when turning it on.